lib/realtime/block_waiver_store.ex

```diff
@@ -27,7 +27,8 @@ defmodule Realtime.BlockWaiverStore do
   def start_link(opts \\ []) do
     name = Keyword.get(opts, :name, default_name())
 
-    GenServer.start_link(__MODULE__, %__MODULE__{}, name: name)
+    # always fullsweep: this keeps us from having the old Schedule data stick around
+    GenServer.start_link(__MODULE__, %__MODULE__{}, name: name, spawn_opt: [fullsweep_after: 0])
```

Comment:
@lemald it turns out Elixir 1.11 has the wrong type for `spawn_opt` :( 

I can either keep us on 1.11 and ignore the Dialyzer error, or I can bump us up to 1.12 or 1.13 (which have the right type): up to you.

Reply:
I was actually just thinking the other day that we should bump the Elixir version in Skate sometime, so if it ends up not causing any other difficulties I'd say go ahead and upgrade.